packages/core-flows/src/payment/steps/capture-payment.ts

```diff
@@ -0,0 +1,22 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IPaymentModuleService } from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+type StepInput = {
+  payment_id: string
+  captured_by?: string
```

Comment:
q: should we be able to pass an amount as an argument for partial capture?

Reply:
es, nice catch. Will add in 👍